### ISSUE TITLE
Fix Accessibility Violations

### ISSUE THREAD

**Issue by srikant-ch5** [elyra-ai/canvas#2191]
Ref: 16314

<img width="645" alt="Screenshot 2024-10-22 at 4 36 35 PM" src="https://github.com/user-attachments/assets/32f280da-19d3-47c1-8a3b-d8ef28f9ecfd">

**Comment by srikant-ch5** [comment]
Closing this issue as there is already an issue related to Accessibility violations.
